code-challenges.test.js

```diff
@@ -17,35 +17,104 @@
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
+  { name: "arthur dent", occupation: "a radio employee" },
+];
+
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
+describe("namer", () => {
+  it("take in an array of objects and returns an array with a sentence about each person with their name capitalized.", () => {
+    expect(namer(people)).toEqual([
+      "Ford Prefect is a hitchhiker",
+      "Zaphod Beeblebrox is president of the galaxy",
+      "Arthur Dent is a radio employee",
+    ]);
+  });
+});
 
```

Comment:
good structure

Reply:
oh my i'm so sorry "banging my head on the desk!"